Python/Product/TestAdapter.Executor/UnitTest/TestExecutorUnitTest.cs

```diff
@@ -805,87 +779,7 @@ out testMethod
             frameworkHandle.RecordResult(result);
             frameworkHandle.RecordEnd(result.TestCase, outcome);
         }
-
-        class TestReceiver : ITestCaseDiscoverySink {
-            public List<TestCase> Tests { get; private set; }
-
-            public TestReceiver() {
-                Tests = new List<TestCase>();
-            }
-
-            public void SendTestCase(TestCase discoveredTest) {
-                Tests.Add(discoveredTest);
-            }
-        }
-
-        sealed class PythonProjectSettings : IEquatable<PythonProjectSettings> {
-            public readonly string ProjectHome, WorkingDirectory, InterpreterPath, PathEnv;
```

Comment:
this deleted code has already been moved to RunSettingsUtil.. just updating this file to use the new stuff